build

```diff
@@ -1,54 +1,63 @@
 #!/bin/bash
-cd /tmp/rom # Depends on where source got synced
+cd /tmp/rom || exit                               # Depends on where source got synced
+#--------------------------EDIT FROM HERE-------------------------------
+#  EDIT IMP VARIABLES AND COLLECT CCACHE BEFORE BUILD ELSE CI TIMEOUT
 
-# Normal build steps
-. build/envsetup.sh
-lunch aosp_mido-user
-export CCACHE_DIR=/tmp/ccache
-export CCACHE_EXEC=$(which ccache)
-export USE_CCACHE=1
-ccache -M 20G # It took only 6.4GB for mido
-ccache -o compression=true # Will save times and data to download and upload ccache, also negligible performance issue
-ccache -z # Clear old stats, so monitor script will provide real ccache statistics
-
-# Next 8 lines should be run first to collect ccache and then upload, after doning it 1 or 2 times, our ccache will help to build without these 8 lines
-#make api-stubs-docs || echo no problem, we need ccache
-#make hiddenapi-lists-docs || echo no problem, we need ccache
-#make system-api-stubs-docs || echo no problem we need ccache
-#make test-api-stubs-docs || echo no problem, we need ccache
-#make aex -j10 & # dont remove that '&'
-#sleep 85m
-#kill %1
-#ccache -s
-#and dont use below codes for first 1 or 2 times, to get ccache uploaded,
-
-
-# upload function for uploading rom zip file! I don't want unwanted builds in my google drive haha!
+export botapi="$mybot"  # add your own bot api                      
+export id="$myid"                                 # Your telegram id check in sangmatabot@tg
+export build_type="final"                         # for {final} build use final or use {ccache} to collect ccache
+export ccache_size="12G"                          # dont use above 10-15 gb dont abuse system
+export ccachedir="/tmp/ccache"                    # ccache dir dont edit if u dunno what ur doing
+export lunchcommand="lunch aosp_mido-userdebug"   # edit as per stated on manifest
```

Comment:
it sometimes seems nice to have a variable like this, but from my point of view its not, i wrote all build commands at a place, any beginner should be able to recognize those.

Reply:
YES IT SHOULD BE BEGINNER FRIENDLY YOU CAN EDIT IT YOUR WAYS IT WOULD BE NICE